tests/testthat/test-bayesianpredictionperformance.R

```diff
@@ -0,0 +1,425 @@
+context("Bayesian Prediction Model Performance")
+
+skip("metamisc does not implement set seed function")
+# O:E: normal/log
+{
+  options <- analysisOptions("BayesianPredictionPerformance")
+  options$chains <- 2
+  options$diagnosticsAcPlot <- TRUE
+  options$diagnosticsGelmanRubinPlot <- TRUE
+  options$diagnosticsRmPlot <- TRUE
+  options$exportColumns <- FALSE
+  options$exportCstat <- "JaspColumn_17_Encoded"
+  options$exportCstatlCI <- "JaspColumn_18_Encoded"
+  options$exportCstatuCI <- "JaspColumn_19_Encoded"
+  options$exportOE <- "JaspColumn_20_Encoded"
+  options$exportOElCI <- "JaspColumn_21_Encoded"
+  options$exportOEuCI <- "JaspColumn_22_Encoded"
```

Comment:
... what? The data doesn't use these as column names? Can't you use `Study` and `c.index` directly?

Reply:
changed to wrapping by ----